test/src/components/Modal-test.js

```diff
@@ -30,12 +25,12 @@ describe('Modal', () => {
   it('should show modal', () => {
     var renderedModal = render(<Modal />).first();
     renderedModal.showModal();
-    renderedModal.elm.modal.called.should.equal(true);
+    renderedModal.state.isShown.should.equal(true);
   });
   it('should hide modal', () => {
     var renderedModal = render(<Modal />).first();
     renderedModal.hideModal();
-    renderedModal.elm.modal.called.should.equal(true);
+    renderedModal.state.isShown.should.equal(false);
```

Comment:
Same here, this is a `hideModal()` test.  When testing methods, you can give them their own describe block and set of it() blocks for what they do.